addons/auth_signup/__openerp__.py

```diff
@@ -17,10 +17,10 @@
         'web',
     ],
     'data': [
-        'auth_signup_data.xml',
-        'res_config.xml',
-        'res_users_view.xml',
-        'views/auth_signup_login.xml',
+        'views/res_config_views.xml',
+        'views/res_users_views.xml',
+        'views/auth_signup_login_templates.xml',
+        'data/auth_signup_data.xml',
```

Comment:
default values for 'installable' and 'version' attributes are 'True' and '1.0', you may omit it.